extensions/machine-learning/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "machine-learning",
   "displayName": "%displayName%",
   "description": "%description%",
-  "version": "0.9.0",
+  "version": "0.10.0",
   "publisher": "Microsoft",
   "preview": true,
   "engines": {
```

Comment:
do we need to change azdata version too?

Reply:
sure I'll do that. I don't think it's needed  every time if it works with older version